docs/tutorials/cloud-foundry.md

```diff
@@ -0,0 +1,730 @@
+---
+layout: default
+title: "Migrating Cloud Foundry apps to Kubernetes"
+permalink: /tutorials/cloud-foundry/
+parent: Tutorials
+nav_order: 4
+---
+
+# Migrating and deploying Cloud Foundry applications to Kubernetes
+
+## Description
+
+This document takes us through the steps that will install Move2Kube and use Move2Kube's 3 step process (collect, plan and transform) to create deployment artifacts for Cloud Foundry apps. It also takes through the process to customize for a specific cluster. Here, we are going to use the data from [samples/cloud-foundry](https://github.com/konveyor/move2kube-demos/tree/main/samples/cloud-foundry).
+
+
+## Prerequisites
+
+1. Install Move2Kube.
+
+   ```console
+   $ MOVE2KUBE_TAG='v0.3.0-rc.0' bash <(curl https://raw.githubusercontent.com/konveyor/move2kube/main/scripts/install.sh)
+   ```
+
+2. Install dependencies.
+  * [Docker](https://www.docker.com/get-started)
+  * [operator-sdk](https://docs.openshift.com/container-platform/4.1/applications/operator_sdk/osdk-getting-started.html#osdk-installing-cli_osdk-getting-started)
+  * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+   To verify that dependencies were correctly installed you can run the below commands.
+   ```console
+   $ operator-sdk version
+   ```
+   ```console
+   $ docker version
+   ```
+   ```console
+   $ kubectl version
+   ```
+
+3. Install [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)
+
+4. Clone the [move2kube-demos](https://github.com/konveyor/move2kube-demos) repository
+
+   ```console
+   $ git clone https://github.com/konveyor/move2kube-demos.git
+   ```
+
+   ```console
+   $ cd move2kube-demos
+   ```
+   Let's see the structure inside the `./samples/cloud-foundry` directory.
+
+   ```console
+   move2kube-demos git:(main) $ tree samples/cloud-foundry
+   samples/cloud-foundry
+   ├── cfapps.yaml
```

Comment:
The CF apps format has changed. Check with @seshapad on the new format and let's update the demos repo

Reply:
okay sure